specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -34,43 +34,91 @@
     }
   },
   "paths": {
-      "/providers/Microsoft.SecurityInsights/operations": {
-        "get": {
-            "operationId": "Operations_List",
-            "description": "Lists all operations available Azure Security Insights Resource Provider.",
-            "parameters": [
-                {
-                    "$ref": "#/parameters/ApiVersion"
-                }
-            ],
-            "produces": [
-                "application/json"
-            ],
-            "responses": {
-                "200": {
-                    "description": "OK. Successfully retrieved operations list.",
-                    "schema": {
-                        "$ref": "#/definitions/operationsList"
-                    }
-                }
-            },
-            "x-ms-pageable": {
-                "nextLinkName": "nextLink"
+    "/providers/Microsoft.SecurityInsights/operations": {
+    "get": {
+      "operationId": "Operations_List",
+      "description": "Lists all operations available Azure Security Insights Resource Provider.",
+      "parameters": [
+        {
+          "$ref": "#/parameters/ApiVersion"
+        }
+      ],
+      "produces": [
+        "application/json"
+      ],
+      "responses": {
+        "200": {
+          "description": "OK. Successfully retrieved operations list.",
+          "schema": {
+            "$ref": "#/definitions/operationsList"
+          }
+        }
+      },
+      "x-ms-pageable": {
+        "nextLinkName": "nextLink"
+      }
+    }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/scheduledAlertRules": {
```

Comment:
In the samples you call this resource 'alertRules' not 'scheduledAlertRules'. Its more concise and I don't think prefixing with 'scheduled' adds any real value.

Reply:
I agree and I started with alertRules, but we understood that we are gonna have soon other alertRules that will behave very differently and they will all create alerts in the system-
compoundAlertRules, realtimeAlertRules, machineLearningAlertRules and few more.
I don't want this endpoint sound to generic because it is a very specific one.
Changed all place to scheduledAlertRules

Reply:
I did so, hopefully I did it right